dora/core/client/fs/src/main/java/alluxio/client/file/DoraFileOutStream.java

```diff
@@ -203,34 +208,60 @@ public void write(byte[] b, int off, int len) throws IOException {
   }
 
   private void writeInternal(int b) throws IOException {
-    if (mWriteToAlluxio) {
-      Integer intVal = b;
-      byte[] bytes = new byte[]{intVal.byteValue()};
-      mNettyDataWriter.writeChunk(bytes, 0, 1);
-      Metrics.BYTES_WRITTEN_ALLUXIO.inc();
-    }
+    try {
```

Comment:
Can we call `writeInternal(byte[] b, int off, int len)` to avoid code duplication?
something like:
```
      Integer intVal = b;
      byte[] bytes = new byte[] {intVal.byteValue()};
      writeInternal(bytes, 0, bytes.length);
```

Reply:
Good catch. Updated.